src/EditorFeatures/CSharpTest/NameTupleElement/NameTupleElementTests.cs

```diff
@@ -23,6 +23,30 @@ public async Task TestInCall_FirstElement()
 @"class C { void M((int arg1, int arg2) x) => M((arg1: 1, 2)); }");
         }
 
+        [Fact]
+        public async Task TestInCall_Deep()
+        {
+            await TestInRegularAndScript1Async(
+@"class C
+{
+    void M((int arg1, int arg2) x) => M((Method([||]1), 2));
```

Comment:
Ok, I thought we wouldn't want to offer the refactoring here because we only cared about parens/commas of the tuple. But if you think it otherwise, that's fine.